.github/ISSUE_TEMPLATE/report_documentation_issue.yml

```diff
@@ -0,0 +1,80 @@
+name: ❓ Report documentation issue
+description: Report problems and issues with Dagster documentation.
+labels: ["documentation", "content-gap"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        <p align="center">
+          <a target="_blank" href="https://dagster.io">
+            <picture>
+              <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/dagster-io/dagster/rl/06-28-docs_use_dagster_brand_colors_for_README/.github/dagster-logo-dark.svg">
+              <img alt="dagster logo" src="https://raw.githubusercontent.com/dagster-io/dagster/rl/06-28-docs_use_dagster_brand_colors_for_README/.github/dagster-logo-light.svg" align="left" width="auto" height="120">
+            </picture>
+          </a>
+        </p>
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for contributing to improve Dagster.
+        If you have a change ready to submit, you do not need to create an issue.
+        Instead, you can open a [pull request](https://github.com/dagster-io/dagster/pulls).
+  - type: markdown
+    attributes:
+      value: >
+        Writer's Guide: [Docs README](https://github.com/dagster-io/dagster/tree/master/docs)
+  - type: textarea
+    attributes:
+      label: What content is missing
+      description: Help us to understand your request in context.
+      placeholder: Please provide a brief description of the missing content.
```

Comment:
Same here, re: `missing content`. Not every doc issue that's reported is about missing content.

Maybe something like: `Please provide a brief description of the issue.`